apps/fluent-tester/src/FluentTester/theme/CustomThemes.ts

```diff
@@ -22,7 +22,7 @@ export const lightnessOptions = [
 ];
 
 export class TesterThemeReference extends ThemeReference {
-  private _themeName: ThemeNames = 'Office';
+  private _themeName: ThemeNames = Platform.OS == ('win32' as any) ? 'Office' : 'Default';
```

Comment:
Uh this was a mistake commit on my part - you can change it back to default for everyone

Reply:
Thanks! I was going to ask about that :D